docs/digger/installation/known-issues.asciidoc

```diff
@@ -3,9 +3,17 @@
 
 === Issues with using HTTP Proxy
 
-==== HTTP proxy with mac Jenkins node
-When adding a mac machine as a node/slave in Jenkins, a direct connection is
-required. This means that a HTTP proxy is not supported for this situation.
+==== HTTP proxy with macOS Jenkins node
+
+When configuring Jenkins to work behind a HTTP proxy, there is a https://issues.jenkins-ci.org/browse/JENKINS-31464[limitation/bug] in the way slaves nodes use the Git plugins as they slave inherits proxy information from the master.
```

Comment:
Grammar or and words in this sentence feels weird to me. I am not a native speaker though :)